benchmark/REQUIRE

```diff
@@ -0,0 +1 @@
+BenchmarkTools

```

Comment:
does this still use REQUIRE?

Reply:
Yes PkgBenchmark still uses REQUIRE to work